_component_design/dropdowns.md

```diff
@@ -10,164 +10,87 @@ preview-image: components/preview-selectors.svg
-  </div>
-</div>
-
-## Type Ahead Multiple Select
-
-<div class="ui form">
-  <div class="field">
-    <select class="ui fluid search dropdown" multiple="">
-      <option value="">State(s)</option>
-      <option value="AL">Alabama</option>
-      <option value="AK">Alaska</option>
-      <option value="AZ">Arizona</option>
-      <option value="AR">Arkansas</option>
-      <option value="CA">California</option>
-      <!-- Saving your scroll sanity !-->
-      <option value="OH">Ohio</option>
-      <option value="OK">Oklahoma</option>
-      <option value="OR">Oregon</option>
-      <option value="PA">Pennsylvania</option>
-      <option value="RI">Rhode Island</option>
-      <option value="SC">South Carolina</option>
-      <option value="SD">South Dakota</option>
-      <option value="TN">Tennessee</option>
-      <option value="TX">Texas</option>
-      <option value="UT">Utah</option>
-      <option value="VT">Vermont</option>
-      <option value="VA">Virginia</option>
-      <option value="WA">Washington</option>
-      <option value="WV">West Virginia</option>
-      <option value="WI">Wisconsin</option>
-      <option value="WY">Wyoming</option>
-    </select>
-  </div>
-</div>
+# Drop-down fields
+
+Drop-downs are a core input mechanism for form and are used to imply and offer
+options or actions.
+
+{% figure [caption:"A generic tab"] [class:"image bg-light border"] %}
+ ![]({{site.url}}/assets/images/components/content-areas/dropdowns/dropdowns-hero.svg){:width="80%"}
+ {% endfigure %}
+
+## When to use drop-downs
+
+Drop-downs are used when a user is required to select a single option from
+a known list of options. For example, selecting a region on the
+Rackspace Cloud server create form will provide the following options:
+
+* Chicago
+* Dallas
+* Hong Kong
+* London
+* Virginia
+
+## Best practice for drop-downs
+
+* Applying the disabled state to any options that are not currently available to
+  the user.
+* Keeping the drop-down pattern to a set of items that are easily parsable by a
+  user. While there is not a hard rule here for an upperbounds limit of items
+  to include a drop-down, once a drop-down data set exceeds 10 items it should
+  be looked at to determine if it could be streamlined.
+* Keeping the drop-down label viewable through all states of the dropdown. This
+  ensures users remember what label prompted the data set they are viewing.
+
+# Specifications
+
+## Default layout
+
+Drop-downs
+
+{% figure [caption:"Default layout"] [class:"image bg-light border"] %}
+ ![]({{site.url}}/assets/images/components/content-areas/dropdowns/dropdowns-default-layout.svg){:width="80%"}
+ {% endfigure %}
+
+## Drop-down states
+
+Drop-downs can be in the following states:
+
+* Default
+* Disabled
+* Focus
+* Error
+
+{% figure [caption:"When you need to describe an image in greater detail, use this block
+ of text so that you can keep the left area clutter-free. 3 lines or less."] [class:"image bg-light border"] %}
```

Comment:
Can kill this caption -- it's just placeholder text that should have been removed before the PDF was exported.